orchestra/static/orchestra/dashboard/tasktable.html

```diff
@@ -5,6 +5,10 @@
     <div class="row section-header">
       <div class="col-lg-10">
         <h3>
+          <span class="collapsed-toggle" ng-click="vm.toggleCollapsed()">
```

Comment:
If it's easy, i'd make the entire row (except the search bar) have the click handler.